python/cudf/bindings/groupby/hash.pxd

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) 2019, NVIDIA CORPORATION.
+
+# cython: profile=False
+# distutils: language = c++
+# cython: embedsignature = True
+# cython: language_level = 3
+
+from libcpp.utility cimport pair
+from libcpp.vector cimport vector
+
+from cudf.bindings.cudf_cpp import *
+from cudf.bindings.cudf_cpp cimport *
+
+cimport cudf.bindings.groupby.common as groupby_common
+
+cdef extern from "groupby.hpp" namespace "cudf::groupby::hash":
+    cdef cppclass Options(groupby_common.Options):
+        pass
+
+    ctypedef enum operators:
+        SUM,
+        MIN,
+        MAX,
+        COUNT,
+        MEAN
+
+    cdef pair[cudf_table, cudf_table] groupby(
+            cudf_table  keys,
+            cudf_table values,
+            vector[operators] ops,
+            Options options
+    )
```

Comment:
Unsure if you left this out to use gdb more easily, but needs to be added before merge 😄 
```suggestion
    ) except +
```